Update readme for AWS Container Insights Receiver

pxaws opened the pull request:
**Description:** 
* update the architecture diagram
* update the metrics and attributes
* add a sample configuration

**Link to tracking Issue:** 
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/2307

pxaws committed:
update readme

pxaws commented:
@Aneurysm9 @anuraaga Could you take a look? Thanks!

Aneurysm9 commented on receiver/awscontainerinsightreceiver/README.md:
```diff
@@ -28,200 +27,646 @@ Note the current implementation is temporarily hosted in AWS OpenTelemetry Colle
+The attribute `container_status_reason` is present only when `container_status` is in "Waiting" or "Terminated" State. The attribute `container_last_termination_reason` is present only when `container_status` is in "Terminated" State.
 
 
+<br/><br/> 
+<br/><br/> 
```
```suggestion
```

Not sure what purpose these serve.

pxaws replied:
Removed

pxaws committed:
Minor changes to address comments

anuraaga reviewed (commented):
This seems like more a design doc than a readme. Can we split it out to a separate doc and make this focused on how to configure the component?

pxaws committed:
split doc to readme and design doc

pxaws commented:
> This seems like more a design doc than a readme. Can we split it out to a separate doc and make this focused on how to configure the component?

It makes sense. I split the doc into readme and design doc.

anuraaga commented on receiver/awscontainerinsightreceiver/README.md:
````diff
@@ -5,223 +5,656 @@ AWS Container Insights Receiver (`awscontainerinsightreceiver`) is an AWS specif
+```
+receivers:
+  awscontainerinsightreceiver:
+    #collection_interval: 60s
````
I don't think we need `#` (if the intent is to indicate they're optional maybe just say in text All parameters are optional)

pxaws replied:
Sure. I will make the change.

anuraaga reviewed (approved):
Thanks, the README is much cleaner. Just one nit

alolita reviewed (approved):
+1
